libraries/cli/include/cli/default_config.hpp

```diff
@@ -204,6 +205,9 @@ const char *default_json = R"foo({
         "genesis_balances": {
           "0274cfffea9fa850e54c93a23042f12a87358a82": "0x141e8d17",
           "111f91441efc8c6c0edf6534970cc887e2fabaa8": "0x24048ce3d"
+        },
+        "hardforks": {
+          "fix_genesis_fork_block": "0x64"
```

Comment:
why can we use normal integers instead of hex? `"fix_genesis_fork_block": 100`

Reply:
Currently we couldn't use decimal ints, because we expecting hex value there. But it is not hard to implement feature as I know

Reply:
but I saw this `json["homestead_block"] = dev::toJS(obj.homestead_block);` so we can parse it same way as eth does. Or I think you can leave it for now, I have created task, to switch everything to readable numbers